### ISSUE TITLE
pathogen#infect causes highlighting to be lost

### ISSUE THREAD

**Issue by kurkale6ka** [neovim/neovim#2015]
Hi,

After upgrading to `NVIM 0.0.0-alpha+201502170110` today, `pathogen#infect` seems to cause highlighting to be lost. If I comment it out, highlighting is back (but I lose my beautiful colors from `csapprox`)

For info: https://github.com/kurkale6ka/vim/blob/master/.vimrc#L12

**Comment by Pyrohh** [comment]
Is the output of `:filetype` the same for both? And is this still an issue now that #2007 has been merged?

**Comment by kurkale6ka** [comment]
`:filetype` is ON for everything and no #2007 doesn't fix it. Just upgraded to `NVIM 0.0.0-alpha+201502182054`

**Comment by kurkale6ka** [comment]
An update on this:

Looks like it's to do with naming the pathogen bundle folder.

Called like this `execute pathogen#infect('.bundle/{}', 'bundle/{}')`,  no syntax:

![synno](https://cloud.githubusercontent.com/assets/52182/6264960/a3e95b62-b820-11e4-9c87-4cf9745efbff.png)

But called like this `execute pathogen#infect('bundle/{}')`, it's ok (except that my colorscheme located in `.bundle` doesn't get parsed):

![syn](https://cloud.githubusercontent.com/assets/52182/6264964/a9da2240-b820-11e4-9627-2faa66d7dcc9.png)

So to sum up, my pathogen custom folder starts with dot and pathogen has support for this but somehow it messes up nvim's syntax highlighting

**Comment by kurkale6ka** [comment]
Note: `pip2 uninstall neovim` doesn't change anything

**Comment by tarruda** [comment]
Is this still a problem on master?

**Comment by kurkale6ka** [comment]
Hi, I tested in xfce4-terminal (tmux or not) and it works even though TERM is screen in tmux. However it doesn't work in xterm with a value of xterm for TERM

**Comment by justinmk** [comment]
The TERM issue is covered elsewhere. The path issue is hopefully fixed.
